grafana/dashboards/cmode/harvest_dashboard_nfs_clients.json

```diff
@@ -83,6 +89,23 @@
     }
   ],
   "panels": [
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "gridPos": {
+        "h": 3,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 29,
+      "options": {
+        "content": "You must enable the Rest Collector in harvest config and template nfs_clients.yaml in $HARVEST/conf/rest/default.yaml.",
```

Comment:
You must enable the Rest collector in your harvest.yml config and uncomment the nfs_clients.yaml section in your $HARVEST/conf/rest/default.yaml file. Please see https://github.com/NetApp/harvest/wiki/FAQ#snapmirror_labels for more details

Reply:
that link is an example, we need to write the REST one and use it 😄